src/PhenX.EntityFrameworkCore.BulkInsert/Extensions/PublicExtensions.DbSet.cs

```diff
@@ -0,0 +1,237 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+using PhenX.EntityFrameworkCore.BulkInsert.Options;
+
+namespace PhenX.EntityFrameworkCore.BulkInsert.Extensions;
```

Comment:
This is the main entry point for the libray. I would move this to the root namespace or even to 'Microsoft.EntityFrameworkCore'